src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,57 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.alive = true;
```

Comment:
also you don`t need this because this redundant, you just setting it false when animal dies